File: 프로그래머스/1/42576. 완주하지 못한 선수/완주하지 못한 선수.java
import java.util.*;
class Solution {
    public String solution(String[] participant, String[] completion) {
        HashMap<String, Integer> list=new HashMap<>();
        for(String str:participant){
            if(!list.containsKey(str)){
                list.put(str, 1);
            }
            else{
                list.put(str, list.get(str)+1);
            }
        }
        for(String str:completion){
            list.put(str, list.get(str)+1);
        }
        for(String str:list.keySet()){
            if(list.get(str)%2==1){
                return str;
            }
        }
        
        return "";
    }
}

File: 프로그래머스/1/42576. 완주하지 못한 선수/완주하지 못한 선수.py
def solution(participant, completion):
    d={}
    for key in participant:
        if key in d:
            d[key]=d[key]+1
        else:
            d[key]=1
            
    for key in completion:
        d[key]=d[key]+1
    for key, value in d.items():
        if value%2==1:
            return key
        
    return ''